app/helpers/show_page_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module ShowPageHelper
+  def convert_solr_value_to_url(value)
+    links = value[:document][value[:field]].map { |link| tag.a link, { href: link, target: '_blank', rel: 'noopener' } }
```

Comment:
If you want to prevent the JS vulnerability here by putting 'noopener', I would also consider putting noreferrer like this  'noopener noreferrer' since noopener is ignored by some of the browsers and noreferrer includes the functionality of noopener.

Reply:
Will change that--thanks for the advice. I didn't really the know the difference and the documentation I read didn't get through to me. That link above helped.

Reply:
Changes are done and passing tests.